### PR TITLE
Update adguard.markdown

### PR DESCRIPTION
**Pull request by mjervelund**
Added note on Certs from let's encrypt mount location.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Doommius, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19131).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/adguard.markdown [hunk 1]
```diff
@@ -109,3 +109,11 @@ Refresh all filter subscriptions in AdGuard Home.
 
 By default, `force` is set to `false`. Forcing an update bypasses AdGuard Home's
 throttling logic, so use with care.
+
+## Encyption
+
+Adguard supports DNS-over-HTTPS and DNS-over-TLS. 
+
+<div class="note">
+When using the Let's encrupt addon the Certs will be mounted at /ssl/ by defaults so the 2 certs can be found at /ssl/fullchain.pem &/ssl/privkey.pem
```
Comment (WesleyDMartin) [inline source/_integrations/adguard.markdown]:
```suggestion
When using the Let's encrypt addon the Certs will be mounted at /ssl/ by defaults so the 2 certs can be found at /ssl/fullchain.pem & /ssl/privkey.pem
```

File: source/_integrations/adguard.markdown [hunk 2]
```diff
@@ -109,3 +109,11 @@ Refresh all filter subscriptions in AdGuard Home.
 
 By default, `force` is set to `false`. Forcing an update bypasses AdGuard Home's
 throttling logic, so use with care.
+
+## Encyption
+
+AdGuard supports DNS-over-HTTPS and DNS-over-TLS. 
+
+<div class="note">
```
Comment (ludeeus) [inline source/_integrations/adguard.markdown:117]: The integration does not handle this. Did you mean to open a change for the add-on?

**Comment by ludeeus** [comment]
Closing as the change here is not related to the integration.

**Pull request closed without merging**